### PR TITLE
[improve] Twitter to Salesforce scenario

### PR DESCRIPTION
**Pull request by lvydra**

### CODE REVIEW

File: ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature [hunk 1]
```diff
@@ -65,6 +68,15 @@ Feature: Create integration with datamapper step
     And scroll "top" "right"
     And click on the "Done" button
 
+    # add a filter step
+    When Camilla clicks on the "Add a Step" button
+    Then Camilla is presented with the "Add a step" link
+    And clicks on the random "Add a step" link
+    And she selects "Basic Filter" integration step
+    And she is presented with a "Basic Filter" step configure page
+    Then she fill configure page for "Basic Filter" step with "ANY of the following, path2, Contains, value2" parameter
```
Comment (dsimansk) [inline ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature]: Let's be more realistic and use something like `text` contains `#salute` or `#syndesis4ever`.

Comment (dsimansk) [reply ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature]: Hashtag part is especially important to mimic Twitter-like behaviour.

Comment (lvydra) [reply ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature]: OK

File: ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature [hunk 2]
```diff
@@ -51,6 +53,7 @@ Feature: Create integration with datamapper step
     # select salesforce connection
     When Camilla selects the "QE Salesforce" connection
     And she selects "Create or update record" integration action
+    And she selects "Contact" from "sObjectNameundefined" dropdown
```
Comment (dsimansk) [inline ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature]: I've a feeling that this dropdown has a proper ID now. Can you pls double check it?
`<select _ngcontent-c25="" class="form-control ng-untouched ng-pristine ng-valid" name="sObjectName" required="" id="sObjectName">`

Comment (lvydra) [reply ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature]: Checked. You are right.

**Pull request merged**